### DISCUSSION TITLE
走路和跑步是混在一起的？

### DISCUSSION THREAD

**Discussion by conge** [yihong0618/running_page#503] [General]
最近开始用手表记录走路和冥想时间
发现在running_page里，我所有的走路，都被记录成了跑步。

比如下面的截图里的第一和倒数第一两条记录，配速大于10，心率低于100的，都是散步而不是跑步。

![image](https://github.com/yihong0618/running_page/assets/1200427/d557abfe-e3ec-40fc-ac00-1eda27b05063)

而同样的记录在Strava里，跑步和散步都是分开的

![image](https://github.com/yihong0618/running_page/assets/1200427/5ae6454b-c100-4ad4-98ec-b0e064495873)

在手表App里，散步和跑步同样是分开的：

![image](https://github.com/yihong0618/running_page/assets/1200427/4ec32300-bcff-4370-b3de-7a6d4132f3cb)

我的running page是从Strava抓数据的。我看了一下抓的数据，第一条我散步的相关数据是这样的：
{"run_id": 9918177238, "name": "Morning Walk", "distance": 1025.4, "moving_time": "0:15:58", "type": "Walk", "start_date": "2023-09-25 15:29:12+00:00", "start_date_local": "2023-09-25 08:29:12", "location_country": "modified to hide it", "summary_polyline": "ez`bHrcviV@g@?YBYC_@?qAB_@j@Ih@@LJDJAJB@\\}A@NIADL", "average_heartrate": 86.9, "average_speed": 1.07, "streak": 187}

我还看了跑步相关的数据，与上面的不同之处是，"type": "Run" 而不是"type": "Walk"。

下次更新时候是不是考虑一下把走路给过滤掉？

**Comment by yihong0618** [comment]
@conge 现在已经可以过滤掉了，--only_run

**Comment by conge** [comment]
谢谢指点。
已经搞定啦
